packages/core/src/balance/Balance.spec.ts

```diff
@@ -8,25 +8,31 @@ import { SubmittableResult } from '@polkadot/api'
 import { GenericAccountIndex as AccountIndex } from '@polkadot/types/generic/AccountIndex'
 import { AccountData, AccountInfo } from '@polkadot/types/interfaces'
 import BN from 'bn.js/'
+import TYPE_REGISTRY from '@kiltprotocol/chain-helpers/lib/blockchainApiConnection/__mocks__/BlockchainQuery'
+import { BlockchainUtils } from '@kiltprotocol/chain-helpers'
 import Identity from '../identity/Identity'
 import {
   getBalance,
   listenToBalanceChanges,
   makeTransfer,
 } from './Balance.chain'
-import TYPE_REGISTRY from '../blockchainApiConnection/__mocks__/BlockchainQuery'
-import { BlockchainUtils } from '../blockchain'
 import BalanceUtils from './Balance.utils'
+import Kilt from '../kilt/Kilt'
 
-jest.mock('../blockchainApiConnection/BlockchainApiConnection')
+// Seems like there is an issue with how this mock is imported or used, does not function.
+// TODO: Look into how the mocked api should be set instead.
```

Comment:
is this still an issue? tests seem to be working.
If there is still a problem we could try jests [auto-mock feature for `node-modules` contents](https://jestjs.io/docs/en/manual-mocks#mocking-node-modules)